### DISCUSSION TITLE
Buen dia

### DISCUSSION THREAD

**Discussion by recarde** [Launch-X-Latam/MisionFrontEnd#442] [Team Posts]
Hola a todos , recien ayer pude entrar al programa y quiero saber cuanto ha avanzado el programa y donde se puede ver las clases anteriores

**Comment by TAGEnrique** [comment]
Hola colega, buenas tardes.
Por lo que tengo entendido, primero debes seguir los pasos del [OnBoarding](https://github.com/Launch-X-Latam/OnBoarding) para que así te agreguen a  un grupo de teams. Ahí encontraras las grabaciones de las sesiones en vivo y demás instrucciones.
Mientras esperas a que te den acceso a Teams, puedes leer el repositorio de [MisionFrontEnd](https://github.com/Launch-X-Latam/MisionFrontEnd). 
Saludos.

**Comment by gumodi35** [comment]
Que tal! 👍 <br>
Entras al Microsoft Teams, si no has entrado dirigete al repositorio **Onboarding** , hay están las instrucciones para ingresar al grupo de Microsoft Teams. <br>
Llenas el form que te dirige al acceso al de Teams y hay tendrás que esperar que te habiliten el acceso al grupo. En ocasiones es al siguiente día, depende.<br>
En el repositorio de la Organización, está el de **Misión frontend** lee, todo hay están las prácticas y hasta ahora solo se ha cubierto hasta HTML, tienes tiempo y no te preocupes si apenas empezaste. <br> 
En el grupo de Teams están las seciones grabadas.<br>
Cualquier cosa mandas mensaje o DM. 💯

**Comment by Mvillablancarodas** [comment]
hola el formulario no esta disponible para ver por lo que no me puedo unir
a teams.. que hago

**Comment by gumodi35** [comment]
Que tal! <br>
Si alguien más comento que tenía problema a la hora del form, trata de intentarlo nuevamente en caso de que no funcione, manda mensaje al insta de innovación, para que te manden el acceso de nuevo, no recuerdo si tiene vigencia o no el lo link del repositorio **Onboarding**  👍

**Comment by Luis-Pedroza** [comment]
Para abrir el formulario, intenten también en una ventana privada y recuerden usar el correo que nos proporcionaron. Me paso lo mismo por tener abierta una cuenta institucional.

**Comment by rpajares** [comment]
Nunca pude llenar ese bendito formulario, nunca cargo o siempre tenia problemas, ahora no se x donde van y que clases son las que dictan, hay mucho desorden

**Comment by gumodi35** [comment]
Don't worry, tienes problema aún, manda DM y te ayudo en lo que necesites! 👍

**Comment by rpajares** [comment]
Hola Gualberto, muchas gracias,  en el team me aparece solo esto

[image: image.png]

no veo nada más, en github que tengo que hacer, donde estan las clases y
que trabajos hay que desarrollar?, porque veo algunos comentarios indicando
que ya terminaron, muchas gracias por tu tiempo.

**Comment by gumodi35** [comment]
dame  unos minutos y te preparo un resumen que te va servir, en breve te lo comparto... :rabbit:

**Comment by Migue-San** [comment]
Va

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by rpajares** [comment]
Perfecto, gracias otra vez.

Le ven. 14 oct. 2022, 13 h 43, Gualberto ***@***.***> a
écrit :

**Comment by gumodi35** [comment]
# Mira asi deberias de trabajar en los espacios de trabajo empecemos con teams: ⭐ 
![teams](https://user-images.githubusercontent.com/80011796/195919498-cef66571-294e-45d5-99e1-f4e42495e277.png) <br>
Aqui vas a entra al grupo de **teams** aqui publican las actividades en los diferentes canales y es en general el medio de comunicacion que mas se usa hay se publican los lives de las seciones. <br>
visita los canales para que veas que se comenta en el dia. <br>
seguimos con el canal de las grabaciones de la primer semana: <br>
![grabaciones](https://user-images.githubusercontent.com/80011796/195919807-7450864e-9a11-4382-89e4-7e9f60ba09bc.png) <br>
Aqui estan las seciones de la 1er Semana de click en la fecha:  🎥 <br>
[26/09/2022](https://web.microsoftstream.com/video/810e53cd-1b58-4297-99e0-222f661dc0cd) 
[28/09/2022](https://web.microsoftstream.com/video/f4309417-2061-4d25-a721-6e8a97625c38)
[30/09/2022](https://web.microsoftstream.com/video/41b9721d-cde1-4ffd-bdf0-7a877d57e5e1)
[05/10/2022](https://web.microsoftstream.com/video/6aaf8bde-6077-4e9b-8db1-cc51db5d29a3)
<br>
2da Semana de grabaciones:  🎥  <br>
[07/10/2022](https://web.microsoftstream.com/video/810e53cd-1b58-4297-99e0-222f661dc0cd) 
[12/10/2022](https://web.microsoftstream.com/video/f4309417-2061-4d25-a721-6e8a97625c38)
<br>
Por ultimo un recorrido en el repositorio, supongo que ya realizaste el Onboarding te recomiendo que lo hagas: 📚 <br>


![repo-launch](https://user-images.githubusercontent.com/80011796/195921804-d98a6f94-0e4b-4a03-a917-662934de93c3.png)

<br>

![intro](https://user-images.githubusercontent.com/80011796/195921870-eef63af3-0acc-4cc8-936a-98b56ef3042a.png)

<br>

![html](https://user-images.githubusercontent.com/80011796/195921889-a7a474c2-8ecb-4cda-872e-c1c33d6972e9.png)

Espero que te sirva y de igual forma estan manda mensaje en los grupos para que te puedan apoyar alguna duda e igual puedes mandarme DM y te voy ayudando! saludos!! 🐰

**Comment by Migue-San** [comment]
Gracias excelente información

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by DanielSilva26** [comment]
Toda la info está en Teams y GitHub compañero 😁

[IMG_7245.png]

Daniel Silva

El 14 oct 2022, a la(s) 12:41 p.m., rpajares ***@***.***> escribió:

﻿

Nunca pude llenar ese bendito formulario, nunca cargo o siempre tenia problemas, ahora no se x donde van y que clases son las que dictan, hay mucho desorden

**Comment by rpajares** [comment]
Excelente, muchas gracias por la info.

**Comment by Mai026** [comment]
Muchas gracias por la info

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
